packages/typespec-python/scripts/run_tsp.py

```diff
@@ -34,7 +36,8 @@
         breakpoint()  # pylint: disable=undefined-variable
 
     # run m2r
-    python_run(venv_context, "generator.pygen.m2r", command=sys.argv[1:])
-    python_run(venv_context, "generator.pygen.preprocess.__init__", command=sys.argv[1:])
-    python_run(venv_context, "generator.pygen.codegen.__init__", command=sys.argv[1:])
-    python_run(venv_context, "generator.pygen.black", command=sys.argv[1:])
+    args, unknown_args = parse_args()
+    m2r.M2R(output_folder=args.output_folder, cadl_file=args.cadl_file, **unknown_args).process()
+    preprocess.PreProcessPlugin(output_folder=args.output_folder, cadl_file=args.cadl_file, **unknown_args).process()
+    black.BlackScriptPlugin(output_folder=args.output_folder, **unknown_args).process()
+    codegen.CodeGenerator(output_folder=args.output_folder, cadl_file=args.cadl_file, **unknown_args).process()
```

Comment:
you need to call codegen first then black, otherwise we don't get any formatting of the outputted code